rpcs3/Input/ds4_pad_handler.cpp

```diff
@@ -542,14 +555,21 @@ void ds4_pad_handler::CheckAddDevice(hid_device* hidDevice, hid_device_info* hid
 
 	if (!GetCalibrationData(ds4Dev))
 	{
+		ds4_log.error("CheckAddDevice: GetCalibrationData failed!");
+		hid_close(hidDevice);
+		return;
+	}
+
+	if (hid_set_nonblocking(hidDevice, 1) == -1)
```

Comment:
If non-blocking fails (although I'm not sure it ever would), could we revert to using hid_read_timeout with a very short timeout period (i.e. 1mS).  Obviously not the ideal scenario.. but it would be nicer behaviour than just not allowing the use of that device at all.

Reply:
Why should it fail?
If it fails then something must be wrong.
It's a bad idea to just rewrite the entire pad handler code for some unlikely error without knowing the cause of that error